templates/aws-vpc.template

```diff
@@ -129,19 +129,19 @@
                     "default": "Public subnet 4 CIDR"
                 },
                 "PublicSubnetTag1": {
-                    "default": "Tag for Public Subnets"
+                    "default": "Tag for public subnets"
                 },
                 "PublicSubnetTag2": {
-                    "default": "Tag for Public Subnets"
+                    "default": "Tag for public subnets"
```

Comment:
__Should rename  this to_ Tag 2 for public subnets

Reply:
Added tag numbers to the labels for all subnet tags (private A, private B, public).